content/archive/_index.md

```diff
@@ -19,7 +19,7 @@ This page contains exercises, test sets and solutions of several contests. You c
 # Other archives
  * [ICPC World Finals Problem archive](https://icpc.global/worldfinals/problems)
  * [Open Kattis Problem Sources](https://open.kattis.com/problem-sources)
- * [Uva Online judge](https://onlinejudge.org/)
+ * [UVa Online Judge](https://onlinejudge.org/ "Hosted by University of Valladolid")
```

Comment:
This will break the link. UVa is the acronym for the University of Valladolid. I don't think we need to add it since we also don't change ICPC to International Collegiate Programming Contest.
```suggestion
 * [UVa Online Judge](https://onlinejudge.org/)
```

Reply:
The reason why I clarified it, is because it may be confused with "UvA" (Universiteit van Amsterdam", especially because there was an error in the capitalisation. :stuck_out_tongue: But we can do without the hover-text :slightly_smiling_face: